presto-parquet/src/main/java/com/facebook/presto/parquet/dictionary/LongDictionary.java

```diff
@@ -31,7 +33,8 @@ public LongDictionary(DictionaryPage dictionaryPage)
         super(dictionaryPage.getEncoding());
         content = new long[dictionaryPage.getDictionarySize()];
         LongPlainValuesReader longReader = new LongPlainValuesReader();
-        longReader.initFromPage(dictionaryPage.getDictionarySize(), dictionaryPage.getSlice().getBytes(), 0);
+        final ByteBufferInputStream in = ByteBufferInputStream.wrap(ByteBuffer.wrap(dictionaryPage.getSlice().getBytes()));
```

Comment:
ditto about:
- removing the `final` variable
- the var args call

Reply:
Done.